utils/lists.ts

```diff
@@ -611,4 +611,9 @@ export const derivativesList: Record<string, string>[] = [
     description: "The first anti-metaverse built on Ethereum",
     url: "https://upsidedownloot.com/",
   },
+  {
+    name: "MekaLoot",
```

Comment:
Please, place your added resource into correct alphabetical order. Thanks.

Reply:
Done it. Does it have conflict with current version?